### PR TITLE
Release v0.3.5 

### PR DESCRIPTION
**Pull request by moreati**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -77,6 +77,8 @@ def long_description():
         'Programming Language :: Python :: 3.8',
         'Programming Language :: Python :: 3.9',
         'Programming Language :: Python :: 3.10',
+        'Programming Language :: Python :: 3.11',
+        'Programming Language :: Python :: 3.12',
```
Comment (dw) [inline setup.py:81]: Changelog does not mention 3.12

**Review by dw — APPROVED** [review]
Just one tiny comment about setup.py 3.12 advertisement vs content of changelog, otherwise utterly heroic work on the module importing stuff! Thank you, completely forgotten Python had recently changed that side of things

File: docs/changelog.rst [hunk 2]
```diff
@@ -15,7 +15,18 @@ Release Notes
...
+`directly from GitHub <https://github.com/mitogen-hq/mitogen/>`_.
+
+v0.3.5 (2024-03-17)
+-------------------
+
+* :gh:issue:`987` Support Python 3.11
+* :gh:issue:`885` Fix :py:exc:`PermissionError` in :py:mod:`importlib` when
+  becoming an unprivileged user with Python 3.x
+* :gh:issue:`1033` Support `PEP 451 <https://peps.python.org/pep-0451/>,
+  required by Python 3.12
```
Comment (moreati) [inline docs/changelog.rst:27]: RST backtick isn't closed

File: setup.py (hunk 1 above)
Comment (moreati) [reply setup.py:81]: It is, on line 28 https://github.com/mitogen-hq/mitogen/pull/1044/files#diff-b7f8a8784d5eff6e6b1b2c2529b831cd86132c49699fe48e0d23c17ad4e2d7deR28, but I've just notcied the previous line will break the rendering due to an unbalanced backtick

Comment (moreati) [reply setup.py:81]: Fixed the link

**Pull request merged**